src/lib/components/DayOpenWaterV2.svelte

```diff
@@ -192,16 +183,14 @@
 				};
 			})
 			.filter((v) => v.amReservations.length > 0 || v.pmReservations.length > 0);
-
-		console.log(buoyGroupings);
 	}
+
+	const boatColWidth = (viewMode) => (viewMode == 'admin' ? 'w-20' : 'w-8');
```

Comment:
if we can avoid functions like this it would be cleaner.

one implementation is directly doing in the element directly ie:
```
<div class="w-8" class:w-20={isAdmin}/>
```

Reply:
In my testing, the direct approach didn't seem to work because both w-8 and w-20 were included, and I think w-8 was taking precedence.  Perhaps it needs to be `<div class:w-8={!isAdmin} class:w-20={isAdmin}/>`?